src/lando/dockerflow/views.py

```diff
@@ -0,0 +1,83 @@
+from django.db import connections
+from django.db.utils import OperationalError
+from django.http import JsonResponse
+from django.views import View
+
+from lando.dockerflow.decorators import disable_caching, log_request
+
+
+class VersionView(View):
+    """
+    This view handles the version information of Lando.
+
+    It returns a JSON response containing the version information.
+    """
+
+    @disable_caching
+    @log_request
+    def dispatch(self, request, *args, **kwargs):
+        return super().dispatch(request, *args, **kwargs)
+
+    def get(self, request):
+        try:
+            from lando.version import version
+
+            data = {
+                "version": version,
+            }
+            status_code = 200
+        except ImportError:
+            data = {"error": "Service Unavailable"}
+            status_code = 503
+
+        return JsonResponse(data, status=status_code, json_dumps_params={"indent": 2})
+
+
+class HeartbeatView(View):
+    """
+    This view handles the heartbeat check which determines if
+    the application is healthy and running.
+
+    It returns a JSON response containing the heartbeat information.
+    """
+
+    @disable_caching
+    @log_request
+    def dispatch(self, request, *args, **kwargs):
+        return super().dispatch(request, *args, **kwargs)
+
+    def get(self, request):
+        db_conn = connections["default"]
+        try:
+            db_conn.cursor()
+            healthy = True
+        except OperationalError:
+            healthy = False
+
+        data = {
+            "healthy": healthy,
+            "services": {
+                "lando": healthy,
+            },
+        }
+
+        status = 200 if healthy else 502
```

Comment:
I don't think `502` is the right status code here.

Reply:
That's what's in [the original](https://github.com/mozilla-conduit/lando/blob/c17b6829e6851381390cf5ccaa3741320c7d4415/src/lando/ui/legacy/dockerflow.py#L83).

I'll switch it to 503, I think that makes more sense.